Person/Program.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace Person
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            
+            string a = "Hello world!";
+            Console.WriteLine(a);
+        }
+    }
```

Comment:
You must edit title for class _Program_, this class should be with title _"Person"_. 
Delete variable _a_ then add integer variable _b_ with value 55. 
@7Woker7

Reply:
I don't care. Do it yourself